client-misk-jooq/src/test/kotlin/app/cash/backfila/client/misk/JooqMenuBackfillDbSetup.kt

```diff
@@ -0,0 +1,64 @@
+package app.cash.backfila.client.misk
+
+import app.cash.backfila.client.misk.jooq.gen.tables.references.MENU
+import app.cash.backfila.client.misk.setup.JooqTransacter
+import okhttp3.internal.toImmutableList
+
+object JooqMenuBackfillDbSetup {
```

Comment:
`FillJooqMenuBackfillDb`? or `JooqMenuBackfillDbDataSetup`

Reply:
Or maybe this should just be a companion on the below test?

Reply:
`JooqMenuBackfillDbDataSetup`